File: lib/ruby/modules.rb
module Stuff
  def self.things
    50
  end

  def self.eat 
    "food"
  end

  def self.puke
    "puke up food"
  end
end

puts Stuff.things
# => 50
puts Stuff.eat
# => "food"
puts Stuff.puke
# => "puke up food"

File: lib/elixir/modules.exs
defmodule Stuff do
  def things do
    50
  end

  def eat, do: "food"
  def puke, do: "puke up food"
end

Stuff.things
# => 50
Stuff.eat
# => "food"
Stuff.puke
# => "puke up food"

# this is the most trivial example of an elixir module
# as you can see it is quite like Ruby's standard module
# syntax

